src/main/java/dao/VolunteerServiceDAOStub.java

```diff
@@ -10,7 +10,8 @@
 @Repository
 public class VolunteerServiceDAOStub implements VolunteerServiceDAO {
 
-    List<Volunteer> allVolunteers = new ArrayList<Volunteer>();
+    //List<Volunteer> allVolunteers = new ArrayList<Volunteer>();
+    Map<Integer, Volunteer> allVolunteers = new HashMap<>();
```

Comment:
Map has different operations than List (put() versus add(), for instance).  I don't see indications that this change included updates to those methods, so this change likely does not compile.  Only code that compiles should be pushed to version control.